.github/workflows/make-build.yaml

```diff
@@ -0,0 +1,31 @@
+name: make build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  web-e2e:
+    name: Frontend e2e test
```

Comment:
emm no

Reply:
Done

Reply:
Emmm.The name doesn't match the behavior.